Ginger/Ginger/ApplicationModelsLib/POMModels/PomElementsPage.xaml.cs

```diff
@@ -106,16 +114,25 @@ public PomElementsPage(ApplicationPOMModel pom, PomAllElementsPage.eElementsCont
             {
                 mElements = mPOM.MappedUIElements;
             }
-            else
+            else if (mContext == PomAllElementsPage.eElementsContext.Unmapped)
             {
                 mElements = mPOM.UnMappedUIElements;
             }
-            
-            SetControlPropertiesGridView();
-            SetLocatorsGridView();
-            SetElementsGridView();
+            else if (mContext == PomAllElementsPage.eElementsContext.AllDeltaElements)
+            {
+                mElements = mPOM.CopiedUnienedList;
+
+                foreach (ElementInfo EI in mElements)
```

Comment:
ElementInfo.eDeltaStatus.Unchanged; should be set by the Wizard before calling the page to show it